optimum/exporters/openvino/model_configs.py

```diff
@@ -1905,7 +1905,7 @@ def get_model_for_behavior(self, model, behavior: Union[str, LlavaNextVideoConfi
         if behavior == LlavaNextVideoConfigBehavior.MULTI_MODAL_PROJECTOR:
             return (
                 model.multi_modal_projector
-                if hasattr(model, "multi_model_projector")
+                if hasattr(model, "multi_modal_projector")
```

Comment:
is it possible to add test(s) to avoid such issues in the future?:)
https://github.com/huggingface/optimum-intel/blob/main/tests/openvino/test_export.py
https://github.com/huggingface/optimum-intel/blob/main/tests/openvino/test_exporters_cli.py